### DISCUSSION TITLE
Data Duplicate_capstone project

### DISCUSSION THREAD

**Discussion by dinaf39** [teamalgoritma/community#658] [Q&A] [answered]
saya sudah melakukan drop duplicate , namun hasil akhirny tidak sama dengan yang ada pada soal. dan sudah saya coba beberapa kali hasilnya tetap beda
![image](https://github.com/teamalgoritma/community/assets/130343206/c4a87c9c-39df-4a8c-84da-5c1005274462)
hasil output akhirnya:
![image](https://github.com/teamalgoritma/community/assets/130343206/d53187cf-011e-460f-b889-c988538a0954)
yang ada di soal:
![Uploading image.png…]()

**Comment by rusdipermana** [comment, marked as answer]
`playstore.drop_duplicates(subset='App', keep='first', inplace=True)`

note : tambahkan  inplace=True
inplace=True adalah sebuah argumen pada metode drop_duplicates() di pandas yang mengindikasikan bahwa operasi penghapusan duplikat harus dilakukan secara langsung pada dataframe yang ada, tanpa perlu menyalin dataframe tersebut ke objek yang baru. Dengan kata lain, argumen inplace=True akan mengubah data yang ada pada dataframe secara permanen, sehingga dataframe asli akan termodifikasi.

Secara default, argumen inplace pada metode drop_duplicates() adalah False, yang berarti bahwa penghapusan duplikat akan menghasilkan dataframe baru dengan duplikat yang dihapus, sedangkan dataframe asli tetap utuh. Jadi, jika inplace=False atau dihilangkan, maka hasil penghapusan duplikat harus disimpan ke variabel lain atau ditugaskan kembali ke variabel yang sama untuk memodifikasinya secara permanen.
